app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from dataclasses import dataclass
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info.get("expiration_date")
+
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine is outdated")
+
+        if "wearing_a_mask" not in visitor or not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("Visitor is not wearing a mask")
```

Comment:
Avoid comparing a variable to `False` or `None` directly. It's more Pythonic to take advantage of the fact that empty containers (like empty lists, empty dictionaries, etc.) and `None` are considered `False` in a boolean context. Here is how you can do it: 

```python
if not visitor.get('wearing_a_mask'):
    raise NotWearingMaskError('Visitor is not wearing a mask')
```